### ISSUE TITLE
Questions about the influence value

### ISSUE THREAD

**Issue by simplelifetime** [princeton-nlp/LESS#21]
I'm wondering what's the appropriate influence value in LESS setting. I'm reproducing it and the max influence value across all sub-tasks are about 0.1-0.4 for some samples. Is this value correct? Or the similarity should be more or less than this value(0.1-0.4 for the most similar sub-task).

**Comment by xiamengzhou** [comment]
Yes, this value is largely within the range of the max gradient similarity for a subtask!
